srcpkgs/ripgrep/template

```diff
@@ -13,6 +13,9 @@ homepage="https://github.com/BurntSushi/ripgrep/"
 distfiles="https://github.com/BurntSushi/${pkgname}/archive/${version}.tar.gz"
 checksum=2513338d61a5c12c8fea18a0387b3e0651079ef9b31f306050b1f0aaa926271e
 
+# Dynamic linking to libpcre2 even on musl
+export PCRE2_SYS_STATIC=0
```

Comment:
Any other Rust stuff using pcre2? We could add this to the Rust helper together with the other variables.